GVFS/GVFS.Common/AzDevOpsFromNuGetFeedUrlParser.cs

```diff
@@ -0,0 +1,41 @@
+using System.Text.RegularExpressions;
+
+namespace GVFS.Common
+{
+    public class AzDevOpsFromNuGetFeedUrlParser
+    {
+        public static bool TryParseOrgFromNuGetFeedUrl(string packageFeedUrl, out string orgName)

```

Comment:
Since these are `static` and will have the class name with the method name, I think we can make these a little shorter and remove the redundancy of `FromNuGetFeedUrl` so you would have  `AzDevOpsFromNuGetFeedUrlParser.TryParseOrg` and `AzDevOpsFromNuGetFeedUrlParser.TryCreate`.  Also the `TryCreate` doesn't feel like it fits if this class is a "parser".

Reply:
I wanted something a little more specific / better than `AzDevOpsNuGetFeedUrlUtils`... `AzDevOrgFromNuGetUrlExtractor`? `AzDevOrgUrlCreator`?